users/test_views.py

```diff
@@ -0,0 +1,8 @@
+class TestViews:
+    def test_profile_view(self, client):
+        response = client.get('/users/profile/')
+        assert response.status_code == 302
+
+    def test_edit_profile_view(self, client):
+        response = client.get('/users/edit_profile/')
```

Comment:
Please use [hyphens](https://pyphilly.org/hyphen-and-underscores-when-use-each-django-and-python/) instead of underscore in url
```suggestion
        response = client.get('/users/edit-profile/')
```